### DISCUSSION TITLE
[FR] Multiple Attributes support

### DISCUSSION THREAD

**Discussion by oerix** [Clooos/Bubble-Card#1100] [Feature requests]
Can we have an option to select more than one attribute for Button entities please?

_Example_: For Media player card, I can choose to have the 'Series name', alongside the 'Channel' name.

The current options `last changed`, `entity state` should all be moved inside the `attribute` dropdown as well, just like HA Tile card shown below. It is much cleaner and easier UI with support for multiple attributes.

**Bubble Card:**
![Screenshot 2024-12-31 at 15-56-13 Overview – Home Assistant](https://github.com/user-attachments/assets/e25e5add-f2ef-412d-9107-6312d8d8fa89)

**HA Tile Card:**
![Screenshot 2024-12-31 at 15-58-20 Overview – Home Assistant](https://github.com/user-attachments/assets/65891666-fff4-46f6-baaa-9a905b00dd3d)

**Comment by Clooos** [comment]
Hi! This feature will finally be the default in all cards and sub-buttons in v3.4.0! 🍻
